strawberry/django/views.py

```diff
@@ -78,21 +77,17 @@ def _render_graphiql(self, request: HttpRequest, context=None):
         if not self.graphiql:
             raise Http404()
 
+        context = context or {}
+        context.update(
+            {"SUBSCRIPTION_ENABLED": "false", "SUBSCRIPTION_GRAPHQL_WS": "false"}
+        )
+
         try:
-            template = Template(render_to_string("graphql/graphiql.html"))
-        except TemplateDoesNotExist:
             template = Template(
-                open(
-                    os.path.join(
-                        os.path.dirname(os.path.abspath(strawberry.__file__)),
-                        "static/graphiql.html",
-                    ),
-                    "r",
-                ).read()
+                render_to_string("graphql/graphiql.html", context=context)
```

Comment:
I think it would be good for consistency if we would let your function handle interpolation here as well. As far as I can tell we do not actually require a custom context nor the posibility to override the graphiql template using the django template system. cc @patrick91 

(If we were to leverage your functions interpolation here as well, we could remove its `raw` parameter which appears to be only used here.)